server/models/handlers/school.js

```diff
@@ -0,0 +1,13 @@
+// Create stub which returns an array of Worker ID's given an input of school ID or school Name
+async function getWorkerIdsForSchool(schoolID) {
+    
+    var array_of_workerIds = false;
+          //* insert (schoolID or schoolName)
+      array_of_workerIds = true;
+          var workerIds=[8000000,8000001,8000002,8000003]
+    return array_of_workerIds;
```

Comment:
Hi Simran,
Based on this, it appears that a value of true will be returned instead of the desired array. I would recommend removing lines 4, 5, and 6 and updating line 8 to return workerIds instead. In addition, please right click on this file and use the Format option available in Visual Studio to properly indent the code and increase code readability.

Reply:
Hi Amy! Thank you! I have added in the changes as required!